apps/src/templates/instructions/ImmersiveReaderButton.jsx

```diff
@@ -32,13 +40,16 @@ class ImmersiveReaderButton extends Component {
 
     return (
       <div
+        tabIndex={0}
+        role="button"
         ref={el => (this.container = el)}
         className={'immersive-reader-button'}
         data-button-style={'icon'}
         data-locale={locale}
         onClick={function() {
           handleLaunchImmersiveReader(locale, title, text);
         }}
+        onKeyDown={e => this.handleKeyDown(e, locale, title, text)}
```

Comment:
nit: Could you avoid having to pass the extra args if you did it like this?

```
onKeyDown={(ev) => {
  if (ev.key === 'Enter' || ev.key === ' ') {
    handleLaunchImmersiveReader(locale, title, text);
    event.preventDefault();
  }
}}
```

Reply:
I could, I find that a bit harder to read though so I'd prefer to leave it factored out

Reply:
Yeah, no worries! The codebase I came from used that style by default.